apps/sms/test/marionette/mocks/mock_navigator_moz_mobile_message.js

```diff
@@ -3,6 +3,7 @@
 
 const Cu = Components.utils;
 Cu.import('resource://gre/modules/Services.jsm');
+Cu.import('resource://gre/modules/Promise.jsm');
```

Comment:
> are you sure?

Well, it definitely didn't work at the time of my comment :) But let me re-check.

> window.Promise 

IMO, it looks worse than ```Cu.import('resource://gre/modules/Promise.jsm');``` - but will check it as well :)